### PR TITLE
Remove stray quote that was causing a 508 launch blocker

### LINKED ISSUE #21786: [AXE-CORE]: VAMC - ARIA attributes must conform to valid values
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix issue.
1. At message the accessibility specialist when issue is resolved.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement
Accessibility specialists discovered Nightwatch axe checks failing silently during a recent staging review. Multiple axe violations were discovered where ARIA attributes were not conforming to valid values. Preliminary research indicates this was mis-formed markup for an `aria-hidden="true"` attribute on right pointing icons. Links to pages below.

<details>
<summary> The violation occurs with the operating status carat </summary>

image

</details

**Element location**
`.vads-u-padding-left--0p5`
**Element source**
<`i class="fa fa-chevron-right vads-u-padding-left--0p5" "="" aria-hidden=" true"></i>`

## Tasks
To solve this violation, you need to fix the following:
- Invalid ARIA attribute value: aria-hidden=" true"
- Remove the extra `"="` markup from the I tag

## Acceptance Criteria
- [ ] All instances of the error are fixed
- [ ] Axe-browser plugin or axe-core CLI scans show 0 Section 508, WCAG2A or WCAG2AA violations

## Environment
* https://www.va.gov/butler-health-care/
* https://www.va.gov/butler-health-care/locations/
* https://www.va.gov/butler-health-care/locations/abie-abraham-va-clinic/
* https://www.va.gov/cheyenne-health-care/
* https://www.va.gov/cheyenne-health-care/locations/
* https://www.va.gov/cheyenne-health-care/locations/cheyenne-va-medical-center/
* https://www.va.gov/cheyenne-health-care/locations/fort-collins-va-clinic/
* https://www.va.gov/cheyenne-health-care/locations/laramie-va-mobile-clinic/
* https://www.va.gov/cheyenne-health-care/locations/loveland-va-clinic/
* https://www.va.gov/cheyenne-health-care/locations/rawlins-va-clinic/
* https://www.va.gov/cheyenne-health-care/locations/sidney-va-clinic/
* https://www.va.gov/cheyenne-health-care/locations/sterling-va-clinic/
* https://www.va.gov/cheyenne-health-care/locations/torrington-va-mobile-clinic/
* https://www.va.gov/cheyenne-health-care/locations/wheatland-va-mobile-clinic/
* https://www.va.gov/erie-health-care/medical-records-office/

## Steps to Recreate
1. Open one of the listed URLs
1. Open axe browser plugin (assumes already installed in Chrome or Firefox)
1. Confirm the `ARIA attributes must conform to valid values` is present one or more times

### PR DESCRIPTION
**Pull request by mpelzsherman**
Resolves https://github.com/department-of-veterans-affairs/va.gov-team/issues/21786

### CODE REVIEW

File: src/site/includes/operatingStatusFlags.drupal.liquid [hunk 1]
```diff
@@ -1,77 +1,40 @@
...
-  onclick="recordEvent({
+  <a class="operating-status-link"
+     onclick="recordEvent({
   'event': 'nav-info-box-click',
   'infoBoxText': 'Facility closed'});"
-  href="{{facilitySidebar.links.0.url.path}}/operating-status"><span
-    class="operating-status-flag operating-status-flag-error operating-status usa-alert usa-alert-error background-color-only vads-u-margin-top--1 vads-u-padding-y--1 vads-u-padding-x--1p5">
-    <i class="fa fa-exclamation-circle" aria-hidden="true"></i> Facility
-    Closed <i
-      class="fa fa-chevron-right vads-u-padding-left--0p5"" aria-hidden="
```
Comment (ncksllvn) [inline src/site/includes/operatingStatusFlags.drupal.liquid:73]: is `--0p5""` the stray quote?

File: src/site/layouts/tests/includes/operatingStatusFlags.unit.spec.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      const container = await renderHTML(layoutPath, data.closed, 'closed');
+
+      expect(
+        container
+          .querySelector('a i.fa-exclamation-circle')
+          .nextSibling.textContent.trim(),
+      ).to.equal('Facility Closed');
+    });
+  });
+});
```
Comment (ncksllvn) [inline src/site/layouts/tests/includes/operatingStatusFlags.unit.spec.js:45]: Fantastic, what a relief to finally have unit tests

**Review by ncksllvn — APPROVED** [review]
LGTM

File: src/site/includes/operatingStatusFlags.drupal.liquid (hunk 1 above)
Comment (mpelzsherman) [reply src/site/includes/operatingStatusFlags.drupal.liquid:73]: yep!

**Review by U-DON — APPROVED** [review]

**Review by brangi — APPROVED** [review]

**Pull request merged**